### DISCUSSION TITLE
the game genie/ram address topic

### DISCUSSION THREAD

**Discussion by Rubiidev-18** [tfdsoft/famidash#137] [General]
talk about codes you found for famidash here!

**Comment by kandowontu** [comment]
Pick any level: Standard Code: 0619:??
?? = level id
![image](https://github.com/famidash/famidash/assets/95717282/c89abd07-015c-49db-9fc0-d1aadbf5be68)

**Comment by kandowontu** [comment]
0604:07 :  Always have all 3 coins collected when finishing level

**Reply by LNLenost** [reply]
Doesnt work anymore

**Comment by Rubiidev-18** [comment]
already did some corruptions on pc, didn't record n stuff tho. (how do I even do something similar to rtc on mobile)
